common/file.c

```diff
@@ -264,7 +264,7 @@ file_read(const char *file, char *buf, size_t len)
 
 	fd = open(file, O_RDONLY);
 	if (fd < 0) {
-		DEBUG("Could not open input file %s", file);
```

Comment:
I changed this because during uevent processing the serial read fails frequently and the log is poluted by this message.
However, it may be better to first check if the serial file exists instead of trying to read it.
Will change this

Reply:
done